buildenv/jenkins/jobs/infrastructure/omrMirror

```diff
@@ -77,4 +77,9 @@ pipeline {
             }
         }
     }
-}
\ No newline at end of file
+    post {
+        failure {
+            slackSend channel: '#jenkins', color: 'danger', message: "Failed: ${env.JOB_NAME} #${env.BUILD_NUMBER} (<${env.BUILD_URL}|Open>)"
```

Comment:
Can you expand on the message?  It will be easier for the people viewing the #jenkins channel if the message is something like "OMR mirror to openj9-omr failed: ${env.JOB_NAME} #${env.BUILD_NUMBER} (<${env.BUILD_URL}|Open>)"

Reply:
@AdamBrousseau pointed out that the job names are reasonably descriptive so the message change isn't necessary